cmd/kubelet/app/server.go

```diff
@@ -625,6 +625,11 @@ func CreateAPIServerClientConfig(s *options.KubeletServer) (*restclient.Config,
 		return nil, err
 	}
 
+	if len(s.ComponentName) > 0 {
+		clientConfig.UserAgent = restclient.UserAgentForComponent(s.ComponentName)
+	} else {
+		clientConfig.UserAgent = restclient.UserAgentForComponent("kubelet")
```

Comment:
Actually, there's const "kubelet" defined in this file: https://github.com/mfojtik/kubernetes/blob/63c20e91f0ed6cae36b637ee1c5f8703abe53b2c/cmd/kubelet/app/server.go#L95